snake_n_ladders/lib/hop.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Hop
+  attr_accessor :start_pos, :end_pos
+
+  def initialize(start_pos, end_pos)
+    @start_pos = start_pos
+    @end_pos = end_pos
+  end
```

Comment:
shouldn't there should be some validations on input?
- type
- value not null
- value > 0